### PR TITLE
Add vbscript documentation and change syntax highlighter

### PR DESCRIPTION
**Pull request by fviale**
- change highlighter to support more languages

### CODE REVIEW

File: src/docs/user/references/VariablesReference.adoc [hunk 1]
```diff
@@ -51,21 +51,21 @@ The order in this list corresponds to the task life-cycle order.
 | -
 | -
 | <<_control_flow_scripts,flow>> (if)
-| bash, cmd, perl
+| bash, cmd, perl, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 2]
```diff
@@ -51,21 +51,21 @@ The order in this list corresponds to the task life-cycle order.
...
 
 | *Parallel runs*. Number of replications. See <<_replicate, replication>>.
 | runs = 3;
 | -
 | -
 | <<_control_flow_scripts,flow>> (replicate)
-| bash, cmd, perl
+| bash, cmd, perl, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 3]
```diff
@@ -51,21 +51,21 @@ The order in this list corresponds to the task life-cycle order.
...
 
 | *Loop variable*. A boolean variable used in a flow script to decide if the loop continues or not. See <<_control_flow_scripts, loop>>.
 | loop = false;
 | -
 | -
 | <<_control_flow_scripts,flow>> (replicate) (loop)
-| bash, cmd, perl
+| bash, cmd, perl, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 4]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
 | -
 | -
 | <<_selection, selection>>
-| bash, cmd, perl
+| bash, cmd, perl, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 5]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
 
 | *Fork Environment*. Fork Environment variable is a link:../javadoc/org/ow2/proactive/scheduler/common/task/ForkEnvironment.html[ForkEnvironment java object] allowing a script to set various initialization parameters of the forked JVM. See <<_fork_environment, Fork Environment>>
 | forkEnvironment.setJavaHome( "/usr/java/default" )
 | -
 | -
 | <<_fork_environment, environment>>
-| bash, cmd, perl, R, PowerShell
+| bash, cmd, perl, R, PowerShell, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 6]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
 
 | *Scheduler API*. Scheduler API variable is a link:../javadoc/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.html[SchedulerNodeClient java object] which can connect to the ProActive Scheduler frontend and interact directly with its <<_task_apis,API>>.
 | schedulerapi.connect()
 | -
 | -
 | <<_fork_environment, environment>>, <<_pre_post_clean, pre>>, <<_script_tasks, task>>, <<_pre_post_clean, post>>, <<_pre_post_clean, clean>>, <<_control_flow_scripts,flow>>
-| bash, cmd, perl, R, PowerShell
+| bash, cmd, perl, R, PowerShell, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 7]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
 
 | *Resource Manager API*. Resource Manager API variable which can connect to the ProActive RM frontend and interact directly with its <<_task_apis,API>>.
 | rmapi.connect()
 | -
 | -
 | <<_fork_environment, environment>>, <<_pre_post_clean, pre>>, <<_script_tasks, task>>, <<_pre_post_clean, post>>, <<_pre_post_clean, clean>>, <<_control_flow_scripts,flow>>
-| bash, cmd, perl, R, PowerShell
+| bash, cmd, perl, R, PowerShell, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 8]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
 
 | *UserSpace API*. UserSpace API variable is a link:../javadoc/org/ow2/proactive/scheduler/task/client/DataSpaceNodeClient.html[DataSpaceNodeClient java object] which can connect to the <<_global_and_user_spaces,User Space>> and interact directly with its <<_dataspace_apis,API>>.
 | userspaceapi.connect()
 | -
 | -
 | <<_fork_environment, environment>>, <<_pre_post_clean, pre>>, <<_script_tasks, task>>, <<_pre_post_clean, post>>, <<_pre_post_clean, clean>>, <<_control_flow_scripts,flow>>
-| bash, cmd, perl, R, PowerShell
+| bash, cmd, perl, R, PowerShell, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 9]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
 
 | *GlobalSpace API*. GlobalSpace API variable is a link:../javadoc/org/ow2/proactive/scheduler/task/client/DataSpaceNodeClient.html[DataSpaceNodeClient java object] which can connect to the <<_global_and_user_spaces,Global Space>> and interact directly with its <<_dataspace_apis,API>>.
 | globalspaceapi.connect()
 | -
 | -
 | <<_fork_environment, environment>>, <<_pre_post_clean, pre>>, <<_script_tasks, task>>, <<_pre_post_clean, post>>, <<_pre_post_clean, clean>>, <<_control_flow_scripts,flow>>
-| bash, cmd, perl, R, PowerShell
+| bash, cmd, perl, R, PowerShell, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc [hunk 10]
```diff
@@ -269,49 +269,49 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
...
 
 | *Synchronization API*. Synchronization API variable is a https://www.activeeon.com/public_content/documentation/javadoc/dev/org/ow2/proactive/scheduler/synchronization/Synchronization.html[Synchronization java object] which can connect to the Synchronization Service and interact directly with its <<_task_synchronization_api,API>>.
 | synchronizationapi.createChannel("channel1", false)
 | -
 | -
 | all scripts
-| bash, cmd, perl, R, PowerShell
+| bash, cmd, perl, R, PowerShell, vbscript
```
Comment (amouhoub) [inline src/docs/user/references/VariablesReference.adoc]: VBScript (as used above in the doc (in other files)

File: src/docs/user/references/VariablesReference.adoc (hunk 1 above)
Comment (fviale) [reply src/docs/user/references/VariablesReference.adoc]: no this is the script engine name, not the wikipedia name

File: src/docs/user/references/VariablesReference.adoc (hunk 10 above)
Comment (amouhoub) [reply src/docs/user/references/VariablesReference.adoc]: @fviale This is what I thought first, but then I was confused by this :
PowerShell, R
They are not script-engine names, so maybe we should fix those

Comment (fviale) [reply src/docs/user/references/VariablesReference.adoc]: R is the name of the script engine, but you are right PowerShell is wrong (wrong case)

Comment (fviale) [reply src/docs/user/references/VariablesReference.adoc]: Fixed for powershell

**Comment by amouhoub** [comment]
@fviale One last comment, do we need to put all those css files ? (especially that they all bare the same class names and that we would be able to use only one theme at a time) shouldn't we only keep the css file (or files) that correspond to the chosen theme instead of adding all of them ?

**Comment by fviale** [comment]
@amouhoub right, I have a look

**Pull request merged**